core/src/main/scala/caliban/schema/Schema.scala

```diff
@@ -683,3 +729,56 @@ trait TemporalSchema {
     )(formatter)
 
 }
+
+case class FieldType(isInput: Boolean, isSubscription: Boolean)
```

Comment:
How about `FieldAttributes`? Just because `Type` is used a lot with a different meaning